docs/sql/statements/copy-from.rst

```diff
@@ -334,6 +334,16 @@ execution.
 The option is only supported when using the ``CSV`` format,
 otherwise, it will be ignored.
 
+``delimiter``
+'''''''''''''
+
+Specifies the character that separates columns within each line of the file.
```

Comment:
this might be better phrased like so:

> Specifies a single one-byte character that separates columns within each line of the file. The default delimiter is `,`.